jobs/openshift/cluster/integreatly/osd-cluster-integreatly-install.yaml

```diff
@@ -92,4 +95,22 @@
                 )
             }  
           }
+          
+          stage('Clone QE repository') {
+            dir('.') {
+              git branch: "master", url: "https://gitlab.cee.redhat.com/integreatly-qe/integreatly-qe.git"
```

Comment:
Do we want to parameterize that? I don't think it's necessary here....

Reply:
hardcoded is fine (if you are referring to the integreatly-qe branch and repo)